frontend/sass/norent/styles.scss

```diff
@@ -369,12 +374,28 @@ $norent-sticky-menu-height: 100px;
 // LETTER BUILDER PAGES
 
 .box.jf-norent-builder-page {
-  // Make container extend just beyond viewport:
-  min-height: calc(100vh - 7rem);
   padding: 4rem 6rem;
   max-width: 700px;
   margin: 4rem auto;
 
+  // These variables estimate the height between the top of the viewport
+  // and the top of a jf-progress-step-wrapper container on desktop and mobile.
+  // The idea is to offset these containers so that the letter builder box extends just beyond the viewport on desktop,
+  // and just before the viewport on mobile (to reveal a sliver of the footer).
+  //
+  // Note: this is NOT an ideal implementation, and should be updated if there are major changes to the sizing
+  // and formatting of the NoRent builder step pages
+  $progress-step-offset: 300px;
+  $progress-step-offset-mobile: 210px;
+
```

Comment:
Agh this is definitely not my best work, but given the complicated nature of how Tahnee wanted the spacing of these containers to behave, it felt like the clearest and easiest solution for our imminent launch. I tried to document the issue as best I could here.

Reply:
Ah cool that's understandable, thanks for documenting it!